core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/tenant/JDBCTenantManager.java

```diff
@@ -936,14 +943,10 @@ public void deleteTenant(int tenantId, boolean removeFromPersistentStorage)
         invalidateCacheManager(domain);
         if (removeFromPersistentStorage) {
             Connection dbConnection = null;
-            PreparedStatement prepStmt = null;
             try {
```

Comment:
let's use try with resources here as well

Reply:
Here also the same as above